index.html

```diff
@@ -0,0 +1,695 @@
+                <p>An avatar is a representation of who we are or our fictional character. </p>
+                <div class="category">
+                    <img src="/assets/Images/avatar1.jpg" class="avatar avatar-sm" />
+                    <img src="/assets/Images/avatar1.jpg" class="avatar avatar-md" />
+                    <img src="/assets/Images/avatar1.jpg" class="avatar avatar-lg" />
+                </div>
+
+                <div class="display-none">
+
+                    <div onclick="showCode('avatars')" class="pre-div">
+                        <i class="fas fa-angle-double-right"></i>
+                    </div>
+                    <div class="pre-tag" id='avatars' style="display: none;">
+                        <iframe src="https://carbon.now.sh/embed/VBdyRItRjCzcDIczEqCw"
+                            style="width: 100%; height: 294px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+
+
+
+            </div>
+            <!-- ALERTS -->
+            <div class="component" id="alerts-id">
+                <h1>Alert</h1>
+                <p class="lead">Alerts are push notifications</p>
+
+                <div class="alert alert-primary">
+                    I am Primary alert
+                </div>
+                <div class="alert alert-secondary">
+                    I am Secondary alert
+                </div>
+                <div class="alert alert-info">
+                    I am Info alert
+                </div>
+                <div class="alert alert-warning">
+                    I am Warning Alert
+                </div>
+                <div class="alert alert-success">
+                    I am Success alert
+                </div>
+                <div class="alert alert-danger">
+                    I am Danger alert
+                </div>
+
+                <div class="display-none">
+
+                    <div onclick="showCode('alerts')" class="pre-div">
+                        <i class="fas fa-angle-double-right"></i>
+                    </div>
+
+                    <div class="pre-tag" id='alerts' style="display: none;">
+                        <iframe src="https://carbon.now.sh/embed/kE2CZFsm0MoxHnCcunHx"
+                            style="width: 100%; height: 510px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+
+
+            </div>
+            <!-- BADGES -->
+            <div class="component" id="badges-id">
+                <h1>Badges</h1>
+                <p class="lead">Badges in Web Design usually helps to highlight certain attributes of an item.</p>
+
+                <div class="category">
+                    <button class="btn btn-google round badge-relative">
```

Comment:
the class round has style background color red, but round indicates something should be round. It would be better if you update round class to e.g. bg-color-red class.
Also btn-google is not a generalized class. please update that.

Reply:
done